nupic/frameworks/opf/clamodel.py

```diff
@@ -219,6 +220,8 @@ def __init__(self,
 
     self.__logger.debug("Instantiated %s" % self.__class__.__name__)
 
+    self._in = None # input data
```

Comment:
More desciptive please. Perhaps `self._input`? And then you don't need the comment?